packages/block-ingestor/src/index.ts

```diff
@@ -0,0 +1,76 @@
+import { PutBlocksReq } from "./api/put-blocks";
+import { BitcoinNetwork, type BlockQueueMessage } from "@gonative-cc/lib/bitcoin";
+
+export async function handleIngestBlocks(request: Request, env: Env): Promise<Response> {
```

Comment:
- move it to a separate file.
- it should not depend on Env. Instead, it should only depend on what it needs: Block store and the Queue